.circleci/config.yml

```diff
@@ -118,6 +118,11 @@ node-integration-base: &node-integration-base
   working_directory: ~/dd-trace-js
   steps:
     - checkout-and-yarn-install
+    - run:
+        name: Install Browser Dependencies
+        command: |
+          apt-get update -y
+          apt install -y fonts-liberation gconf-service libasound2 libatk1.0-0 libcairo2 libcups2 libfontconfig1 libgbm-dev libgdk-pixbuf2.0-0 libgtk-3-0 libicu-dev libjpeg-dev libnspr4 libnss3 libpango-1.0-0 libpangocairo-1.0-0 libpng-dev libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 xdg-utils
```

Comment:
Are these really all necessary?

Reply:
sadly yes for running against a real browser

Reply:
Isn't there any one package that includes all or most of them? My experience with that many requirements to do only one thing is that it keeps breaking as the dependencies change over time.